articles/connections/passing-parameters-to-idps.md

```diff
@@ -0,0 +1,127 @@
+# Passing Parameters to Identity Providers
+
+For Identity Providers that support parameters, you can pass provider-specific parameters to the Authorization endpoint by configuring either static values per connection or dynamic values per user.
+
+Only [valid OAuth 2.0/OIDC parameters](http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint) are accepted.
+
+Some Identity Providers do not support parameters. Check your desired Identity Provider's documentation to see if parameters are supported.
+
+## Passing Static Parameters
+
+To pass static parameters per connection, you can use the `upstream_params` element of the `options` attribute when configuring a connection via the Management API. Each time your connection is used, the specified parameters will be sent to the Identity Provider.
+
+As an example, let's use WordPress, which allows you to pass an optional `blog` parameter to its OAuth 2.0 authorization endpoint (for more information, see [WordPress's OAuth 2.0 documentation](https://developer.wordpress.com/docs/oauth2/)). Let's assume that you want to always request that users have access to the `myblog.wordpress.com` blog when logging in using WordPress.
+
+To accomplish this, you will map WordPress's `blog` parameter to the existing accepted parameter of `access_type` and assign it a default value of `myblog.wordpress.com`:
+
+```
+{
+  "name": "WordPressConn",
+  "strategy": "wordpress",
+  "options": {
+    "upstream_params": {
+      "blog": {
+        "alias": "access_type",
+        "value": "myblog.wordpress.com"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
```

Comment:
looks like you are missing the `Bearer` header (where the token goes, that authenticates with the API).

I think the headers should be like that:
```
"headers": [{
    "name": "Authorization",
    "value": "Bearer YOUR_ACCESS_TOKEN"
  }, {
    "name": "Content-Type",
    "value": "application/json"
  }],
```